gladly-team/tab
# [TFAC-531] Support deprecating background images

* Handle updating users' background images when their current one is deprecated
* Add a new background image, "Solitude"
* Change the default background image for new users

To test:
1. Set your `user.backgroundImage` to have fields:
```
    "id": "9308b921-44c7-4b4e-845d-3b01fa73fa2b", // this image is deprecated
    "thumbnail": "abc.jpg",
    "image": "def.jpg",
```
2. Open a tab
3. Your background image should be "Solitude"
